packages/webamp/js/skinSelectors.ts

```diff
@@ -25,9 +25,6 @@ export const imageSelectors: Selectors = {
   MAIN_STOP_BUTTON_ACTIVE: [".actions #stop:active"],
   MAIN_NEXT_BUTTON: [".actions #next"],
   MAIN_NEXT_BUTTON_ACTIVE: [".actions #next:active"],
-  MAIN_EJECT_BUTTON: ["#eject"],
-  MAIN_EJECT_BUTTON_ACTIVE: ["#eject:active"],
-  MAIN_WINDOW_BACKGROUND: ["#main-window"],
```

Comment:
Long term, this change would let us replace these ~310 lines of random looking CSS selectors with calls to `useSprite` in each component that uses a sprite, thus allowing the reader of the component to know how it gets styled without having to know to check this file.